packages/terra-slide-group/src/Slide.jsx

```diff
@@ -29,6 +30,12 @@ const propTypes = {
    * The aria label for the Slide.
    */
   slideAriaLabel: PropTypes.string,
+
+  /**
+   * @private
```

Comment:
Question: Why is this a private method?

Reply:
Good point. When I originally added this, I was under the impression that Slide would possibly be consumed externally, but it is only used as an internal component within SlideGroup. Therefore, I can remove this tag since the props for Slide are all essentially "private" anyway and never exposed to clients via the docs.